integration/examples/nodejs/skaffold.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: skaffold/v1alpha4
+kind: Config
+build:
+  artifacts:
+  - image: gcr.io/k8s-skaffold/node-example
+    context: backend
+    sync:
+      '*.js': .
+deploy:
+  kubectl:
+    manifests:
+    - "**/k8s"
```

Comment:
I think you have this backwards. You want `k8s/**`

Reply:
this path is relative to the skaffold.yaml

Reply:
You're right! This is a copy-paste from https://github.com/GoogleContainerTools/skaffold/blob/master/integration/examples/hot-reload/skaffold.yaml#L16, so you may want to take a look there too.

Reply:
I changed this to something more explicit